### PR TITLE
fix: fix flake8 errors

### PR DESCRIPTION
**Pull request by sauyon**

### CODE REVIEW

File: tests/integration/frameworks/paddle/test_paddle_runner.py [hunk 1]
```diff
@@ -5,10 +5,11 @@
 
 import bentoml.paddle
 
-from .test_paddle_impl import test_df, train_paddle_model
+# for review: ???
+from .test_paddle_impl import test_df, train_paddle_model  # noqa: F401
```
Comment (sauyon) [inline tests/integration/frameworks/paddle/test_paddle_runner.py:9]: Why do these model imports exist?

File: tests/integration/frameworks/test_pycaret_impl.py [hunk 2]
```diff
@@ -38,7 +38,8 @@ def get_pycaret_data():
 def pycaret_model(get_pycaret_data) -> t.Any:
     # note: silent must be set to True to avoid the confirmation input of data types
     train_data, _ = get_pycaret_data
-    env = pycaret_setup(data=train_data, target="default", session_id=123, silent=True)
+    # for review
+    pycaret_setup(data=train_data, target="default", session_id=123, silent=True)
```
Comment (sauyon) [inline tests/integration/frameworks/test_pycaret_impl.py:42]: Is this change ok?

File: tests/utils/frameworks/tensorflow_utils.py [hunk 3]
```diff
@@ -12,7 +12,8 @@ def __init__(self, units=32, **kwargs):
         super(CustomLayer, self).__init__(**kwargs)
         self.units = tf.Variable(units, name="units")
 
-    def call(self, inputs, training=False, **kwargs):
+    # for review
+    def call(self, inputs, training=False):
```
Comment (sauyon) [inline tests/utils/frameworks/tensorflow_utils.py:16]: Are the `kwargs` for call compatibility? Should I add a pylint ignore here instead?

**Comment by sauyon** [comment]
I actually did a bit of rewording and grammar fixes for documentation but gave up halfway through since I realized some of it may not end up actually used.

Also, documentation reflowing may have caused some format issues in the output; I'm assuming for now that's not a big deal.

File: docker/manager.py [hunk 4]
```diff
@@ -712,9 +712,10 @@ class attributes:
 
         - _push_context (:obj:`Dict`):
 
-            contains mapping for docker.Images object from docker-py with correct releases type
+            contains mapping for docker.Images object from docker-py with correct
+            releases type
 
-    """
+    """  # noqa: LN001
```
Comment (sauyon) [inline docker/manager.py:718]: Is this file currently used? It seems like it doesn't even evaluate without errors.

Comment (aarnphm) [reply docker/manager.py:718]: we are using everything related under docker for creating releases for our docker image.

File: tests/integration/frameworks/paddle/test_paddle_runner.py (hunk 1 above)
Comment (aarnphm) [reply tests/integration/frameworks/paddle/test_paddle_runner.py:9]: i don't like having multiple conftest under smaller directory. It is just a way to write test quickly once I created a model.

Comment (sauyon) [reply tests/integration/frameworks/paddle/test_paddle_runner.py:9]: Oh, these are test fixtures, aren't they.

Comment (aarnphm) [reply tests/integration/frameworks/paddle/test_paddle_runner.py:9]: yes

Comment (sauyon) [reply tests/integration/frameworks/paddle/test_paddle_runner.py:9]: Hm, seems like the recommended way to make these lint is to add a `conftest.py`. Is there any particular reason you don't like having more than one conftest?

The alternative seems to be that or what I've done here which seems a bit ugly too.

File: docker/manager.py (hunk 4 above)
Comment (sauyon) [reply docker/manager.py:718]: Hm, do I have the wrong docker version or something? Flake8 [complains](https://github.com/bentoml/BentoML/runs/4115558301?check_suite_focus=true) that `docker.DockerClient` and `docker.errors` don't exist

Comment (parano) [reply docker/manager.py:718]: @sauyon it seems to work with the latest version of docker python library 5.0.3, but for version 4.x, `docker.DockerClient` and `docker.errors` don't exist

File: tests/utils/frameworks/tensorflow_utils.py (hunk 3 above)
Comment (parano) [reply tests/utils/frameworks/tensorflow_utils.py:16]: it should be ok to remove `**kwargs` here

**Comment by parano** [comment]
@sauyon merging now to avoid more conflicts in other dev branches, let's discuss those questions and address them in follow-up if needed

File: docker/manager.py (hunk 4 above)
Comment (aarnphm) [reply docker/manager.py:718]: Could be because pylint detect the docker at our top level modules instead of the actual docker library.

Comment (parano) [reply docker/manager.py:718]: @aarnphm yes I got rid of that error by removing the `__init__.py` file under the `docker` directory

**Pull request merged**